F4MPBot/cogs/GithubManager.py

```diff
@@ -0,0 +1,85 @@
+from github import Github
+import discord
+import json
+import logging
+import os
+from discord.ext import commands, menus
+
+COG_DIR = os.path.dirname(os.path.abspath(__file__))
+
+GithubBot = Github("Insert Github Personal Access Token (needs read and write permissions for org)")
+
+
+class GithubCommands(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.log = logging.getLogger(__name__)
+
+    @commands.command(brief='<Project Column Name>',
+                      description='This command is used for looking at F4MP Github project tasks. \n Note: May need '
+                                  'to use quotations for multiword args (i.e "Multi Word Arg")')
+    @commands.has_role('Contributor')
+    async def ProjectTasks(self, ctx, args=""):
+        print(args)
+        if (args in ["In progress", "in Progress", "in progress", "In Progress"]):

```

Comment:
Remove redundant parentheses around the condition and replace the List with a tuple as it does not need to be mutable. I also suggest making "args" lowercase and remove all spaces in it then check for if args == "inprogress" to limit issues where someone gives the answer iN progress and it failes.

This also applies to line 31, 38, 45, 60, 65, 70, 75